src/walletConnect/index.tsx

```diff
@@ -95,32 +96,46 @@ export function maybeGoBackAndClearHasPendingRedirect({ delay = 0 }: { delay?: n
  */
 let syncWeb3WalletClient: Awaited<ReturnType<(typeof Web3Wallet)['init']>> | undefined;
 
-const walletConnectCore = new Core({ projectId: WC_PROJECT_ID });
-
-const web3WalletClient = Web3Wallet.init({
-  core: walletConnectCore,
-  metadata: {
-    name: '🌈 Rainbow',
-    description: 'Rainbow makes exploring Ethereum fun and accessible 🌈',
-    url: 'https://rainbow.me',
-    icons: ['https://avatars2.githubusercontent.com/u/48327834?s=200&v=4'],
-    redirect: {
-      native: 'rainbow://wc',
-      universal: 'https://rnbwapp.com/wc',
-    },
-  },
-});
+let lastConnector: string | undefined = undefined;
+
+let walletConnectCore: TypeCore | undefined;
```

Comment:
#nit use `ReturnType<typeof Core>` instead of importing the type from that internal path

Reply:
Tried that but the linter complains:

```
src/walletConnect/index.tsx:100:35 - error TS2344: Type 'typeof Core' does not satisfy the constraint '(...args: any) => any'.
  Type 'typeof Core' provides no match for the signature '(...args: any): any'.
  ```

Reply:
nvm just found the right type!